src/styles/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  padding: 10px;
+  width: $cell-size * 7 + $cell-gap * 6;
```

Comment:
suggestion: you can make new variable $calendar-size and give to it result of this calculation $cell-size * 7 + $cell-gap * 6; After that just use $calendar-size in width

Reply:
Agree, check @serentinos comment
Will be better to move calculation to separate function or create separate variable with this value